pulsar_data_collection/pulse.py

```diff
@@ -0,0 +1,37 @@
+from config import factories
+from models import DataWithPrediction, PulseParameters
+
+
+class Pulse:
+    """
+    This class expose methods in order to collect data from
+    an inference container/webapp
+    """
+
+    def __init__(self, data: PulseParameters):
+        """
+        Initializing Pulse class instance
+        """
+        self.model_id = data.model_id
+        self.model_version = data.model_version
+        self.reference_data_storage = data.reference_data_storage
+        self.target_name = data.target_name
+        factory = factories.get(data.storage_engine)
+        self.storage_engine = factory.get_database_actions()
+        self.db_connection = self.storage_engine.make_connection(**data.login)
```

Comment:
If make_connection methods fail because of the wrong URL, token or bucket info, How do you handle it